### PR TITLE
Manuscript and Appendices read over

### PR DESCRIPTION
**Pull request by mlap**
No structural edits here, mainly grammar/flow/consistency edits.

### CODE REVIEW

**Comment by cboettig** [comment]
@mlap mostly looks good.  Note that in #37 I fixed a few figure references that had the wrong figure numbers, which creates some merge conflicts here.  Can you resolve and merge when ready?

File: appendices/appendix-B-examples.Rmd [hunk 1]
```diff
@@ -242,7 +244,7 @@ if(!file.exists("../python/cache/td3_untuned.zip")) {
 ```
 
 
-Note that while default hyper-parameters provide a useful starting place (particularly when the environment has been suitably normalized, a best-practice we discuss above), better performance can almost always be achieved by _tuning_ the hyper-parameter selection.
+Note that while default hyperparameters provide a useful starting place (particularly when the environment has been suitably normalized, a best-practice we discuss below), better performance can almost always be achieved by _tuning_ the hyperparameter selection. <!-- ML: eek I think we dropped normalization talk -->
```
Comment (mlap) [inline appendices/appendix-B-examples.Rmd:247]:
```suggestion
Note that while default hyperparameters provide a useful starting place (particularly when the environment has been suitably normalized, a best-practice we discuss below), better performance can almost always be achieved by _tuning_ the hyperparameter selection.
```

File: manuscript/references.bib [hunk 2]
```diff
@@ -1881,3 +1881,5 @@ @article{punt_management_2016
 	year = {2016},
 	pages = {303--334},
 }
+
+
```
Comment (mlap) [inline manuscript/references.bib:1885]:
```suggestion
```

File: manuscript/references.bib [hunk 3]
```diff
@@ -1881,3 +1881,5 @@ @article{punt_management_2016
 	year = {2016},
 	pages = {303--334},
 }
+
```
Comment (mlap) [inline manuscript/references.bib:1884]:
```suggestion
```

File: manuscript/manuscript.Rmd [hunk 4]
```diff
@@ -369,7 +376,7 @@ plot_sims(tip_sims_df) / ( plot_policy(tip_policy_df) + plot_reward(tip_sims_df)
 
 The TD3-trained agent proves far more successful in preventing chance transitions across the tipping point, consistently achieving a higher cumulative ecosystem service value across replicates than the steady-state strategy.
 
-Examining the replicate management trajectories and corresponding rewards [Fig 5] reveal that the RL agent incurs significantly higher costs in the initial phases of the simulation, dipping the cumulative reward into negative territory on average and well below the reward realized by the simple steady-state strategy.
+Examining the replicate management trajectories and corresponding rewards [Fig 5] reveal that the RL agent incurs significantly higher costs in the initial phases of the simulation, dipping the cumulative reward into negative territory on average and well below the reward realized by the simple steady-state strategy. <!-- ML: Nit picky; it looks like it almost goes negative but not quite on average in the plot -->
```
Comment (cboettig) [inline manuscript/manuscript.Rmd:379]: good catch, I think didn't revisit this text after I retrained the algo.  Let's just reword this to "dipping well below the mean steady-state reward initially before exceeding it in the long run"  or something like that.

**Review by cboettig — COMMENTED** [review]
good catch on the conservation figure not actually being negative, want to just tweak the text as discussed above?  Otherwise this looks all set, nice work!

**Comment by mlap** [comment]
made those changes @cboettig so going to merge here
